plugins/exporter.koplugin/main.lua

```diff
@@ -1,431 +1,91 @@
-local BD = require("ui/bidi")
-local InputContainer = require("ui/widget/container/inputcontainer")
-local InfoMessage = require("ui/widget/infomessage")
-local NetworkMgr = require("ui/network/manager")
+--[[
+    Export highlights to different targets.
```

Comment:
This is a code block; unlikely to be desired. (Check with `ldoc .`, or maybe there's a kodev command for it these days.)

Reply:
Checked right now. It doesn't get catched by ldoc, unless it is in the form of

```lua
--[[--

Title

Description

@somemodule
]]
```